bfd/elfxx-riscv.c

```diff
@@ -1770,6 +1770,40 @@ riscv_parse_add_implicit_subsets (riscv_parse_subset_t *rps)
 			      RISCV_UNKNOWN_VERSION,
 			      RISCV_UNKNOWN_VERSION, TRUE);
     }
+
+  if ((riscv_lookup_subset (rps->subset_list, "b", &subset)))
```

Comment:
Same as what I mentioned in the K's PR, but we don't need to care about this here.

Reply:
Okay, I keep it same as K defined, no expands.